Userland/Services/WebDriver/Session.h

```diff
@@ -41,6 +41,12 @@ class Session {
     ErrorOr<void, Web::WebDriver::Error> check_for_open_top_level_browsing_context_or_return_error();
     String const& current_window_handle() { return m_current_window_handle; }
 
+    WebContentConnection& web_content()
```

Comment:
I think this should be called `web_content_connection()`, plain getters should pretty much always match their member names